### DISCUSSION TITLE
[Issue] Missing and wrong Taxes

### DISCUSSION THREAD

**Discussion by DoM1niC** [Blair2004/NexoPOS#1329] [Ideas]
**Describe the bug**
Receipt need to clarify two Taxes on Receipt.

**To Reproduce**
1. Setup two Taxes Groups with different Rates in my Case in Germany we split Products in 7% and 19% VAT
2. Setup Product and set the Taxes Group
4. Create a Receipt, in my case only 7% Tax will show but the Invoice are right and calculate 19%. 

I use under VAT Settings "Products & flexible Rate, Products VAT Option doesn't show any Taxes on my Receipt.

**Expected behavior**
If you have more then one Taxgroups so this have to show as Position on Receipt and Invoice.

**Screenshots**
![image](https://user-images.githubusercontent.com/1436958/211168284-693a13cc-0a39-473e-a5e6-8805c711db59.png)
## Example
![image](https://user-images.githubusercontent.com/1436958/211168195-5000eb30-164b-4be1-9c12-4aea7dcc193b.png)

**Environment:**
 - [NexoPOS 4.8.12](https://my.nexopos.com/en)

**Comment by Blair2004** [comment]
Hi,
We appreciate your contribution. We noticed that information is missing on the default receipt (but available on the receipt issued by Nexo Print Adapter). We'll make sure to add that on the web receipt as well.

Thank you.

**Reply by mbugikp** [reply]
I support this feature addition on web receipt. Do we have a timeframe for implementation. 
Regards.
Mbugi
